### ISSUE TITLE
I can't retrieve an existing customer

### ISSUE THREAD

**Issue by stoufly** [flo-sch/stripe-bundle#4]
Can you add this method please ?
/**
     * Retrieve a Customer instance by its ID
     *
     * @throws HttpException:
     *     - If the customerId is invalid
     *
     * @see https://stripe.com/docs/api#customer
     *
     * @param string $customerId: The customer ID
     *
     * @return Customer
     */
    public function retrieveCustomer($customerId){
        return Customer::retrieve($customerId);
    }

**Comment by flo-sch** [comment]
Fixed by 89ab2eea6f28ec2535af840a6b0a77533f7cfbdd, release 0.1.8

**Comment by stoufly** [comment]
Thanks a lot !
But I found a new issue.
I can't do a subscription with an existing customer.

Can you add this method please ?

public function createSubscription($planId, $customerId, $timestampStart=null,  $couponId = null, $tva=null){
        $data = [
            'customer' => $customerId,
            'plan' => $planId,
            'tax_percent'=>$tva
        ];
        if ($couponId) {
            $data['coupon'] = $couponId;
        }
        if($timestampStart){
            $data['trial_end'] = $timestampStart;
            $data['prorate'] = FALSE;
        }
        
        $subscription = Subscription::create($data);
        return $subscription;
    }

Thanks

**Comment by flo-sch** [comment]
Hey @stoufly, alright I'll add this one,

Just so you do not get stucked if you need new helpers quickly, you can always extends "my" stripe client and add your own methods ;)

```
// src/YourNamespace/YourProject/Stripe/StripeClient.php
<?php

namespace YourNamespace\YourProject\Stripe;

use Flosch\Bundle\StripeBundle\Stripe\StripeClient as BaseStripeClient;

class StripeClient extends BaseStripeClient
{
    public function __construct($stripeApiKey = '')
    {
        parent::__construct($stripeApiKey);

        return $this;
    }

    public function createSubscription($planId, $customerId, $timestampStart=null, $couponId = null, $tva=null)
    {
        $data = [
            'customer'      => $customerId,
            'plan'               => $planId,
            'tax_percent'  => $tva
        ];

        if ($couponId) {
            $data['coupon'] = $couponId;
        }

        if ($timestampStart) {
            $data['trial_end'] = $timestampStart;
            $data['prorate']   = false;
        }

        return Subscription::create($data);
    }
}
```

**Comment by stoufly** [comment]
Thanks a lot for your reactivity :)

**Comment by flo-sch** [comment]
Fixed by 35d7e43ceb2eb41bb371fce51aafa041f7116921, release 0.1.9

Now you can do something like :

``` php
$startAt = new \DateTime('2018-01-01 00:00:00');

$stripeClient->subscribeExistingCustomerToPlan($customerId, $planId, [
    'coupon'         => $couponId,
    'tax_percent'    => $taxPercent,
    'trial_end'      => $startAt->format('U'),
    'prorate'        => false
]);
```

3-rd parameter is an associative array which can include any parameter accepted by the Stripe API, you can find the list here : https://stripe.com/docs/api#create_subscription

**Comment by stoufly** [comment]
Great ! Thanks a lot for your help :)
